<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dmMode</class>
 <widget class="QDialog" name="dmMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>561</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>DM Modes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_0">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>95</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="channel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>561</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>dmXXdispYY</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_0">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>105</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_0">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 00</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>210</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 02</string>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>215</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>205</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>265</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>260</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 03</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>255</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>315</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>310</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 04</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>305</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_5">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>360</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 05</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>355</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>365</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>405</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_6">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>410</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 06</string>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>415</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>455</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>465</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_7">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>460</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 07</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_8">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>510</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 08</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>505</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>515</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_9">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>561</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 09</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_9">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>556</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_9">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>566</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_10">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>605</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>615</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_10">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>610</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 10</string>
   </property>
  </widget>
  <widget class="QLabel" name="modeCurrent_1">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>0.0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QwtSlider" name="modeSlider_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lowerBound">
    <double>-1.000000000000000</double>
   </property>
   <property name="upperBound">
    <double>1.000000000000000</double>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="scalePosition">
    <enum>QwtSlider::NoScale</enum>
   </property>
  </widget>
  <widget class="QLabel" name="modeName_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>155</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode 01</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_0">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>95</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_1">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>150</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_2">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>205</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_4">
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>310</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_5">
   <property name="geometry">
    <rect>
     <x>468</x>
     <y>360</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_6">
   <property name="geometry">
    <rect>
     <x>467</x>
     <y>410</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_7">
   <property name="geometry">
    <rect>
     <x>467</x>
     <y>455</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_8">
   <property name="geometry">
    <rect>
     <x>466</x>
     <y>505</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_9">
   <property name="geometry">
    <rect>
     <x>467</x>
     <y>555</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="modeTarget_10">
   <property name="geometry">
    <rect>
     <x>466</x>
     <y>605</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
